ksql-functional-tests/src/test/resources/historical_plans/array_-_GENERATE_SERIES/5.5.0_1581572083765/spec.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
These are being checked in under `/5.5.0_1581572083765/`. Two questions:

1. Is this the 0.7.0 release, or some commits after that?
2. How are people supposed to know this is the 0.7.0 release?

Reply:
> Is this the 0.7.0 release, or some commits after that?
some commits after

> How are people supposed to know this is the 0.7.0 release?
This is a good point (though technically 0.8). Let me think about what makes sense here. In the meantime I'm going to merge this because it's blocking me.